pkg/framework/host.go

```diff
@@ -362,54 +358,80 @@ func (h *Host) InstallOperator(name, cr, values, version string) error {
 	return nil
 }
 
-func (h *Host) InstallResource(name, values, version string, conditions ...func() error) error {
-	chartValuesEnv := os.ExpandEnv(values)
-
-	tmpfile, err := ioutil.TempFile("", name+"-values")
-	if err != nil {
-		return microerror.Mask(err)
-	}
-	defer os.Remove(tmpfile.Name())
-
-	if _, err := tmpfile.Write([]byte(chartValuesEnv)); err != nil {
-		return microerror.Mask(err)
+// InstallResource installs a chart
+// quay.io/giantswarm/${name}-chart:${version}. If the version is prefixed with
+// colon it will be stripped. The first encurtered error from condidtionFns (if
+// any) will be returned.
+func (h *Host) InstallResource(ctx context.Context, name, values, version string, conditionFns ...func() error) error {
```

Comment:
I appreciate the effort you put in here, it would have been nicer to put that effort into switching over to the `resource` package instead IMO.

Reply:
Oh 🔣 . I totally forgot about this package after time spent with aws-operator.